hack/runtests.sh

```diff
@@ -20,7 +20,7 @@
 
 
 set -o errexit
-set -o nounset
```

Comment:
@sgandon any specific reason to remove this?

Reply:
Yes, as I mentionned in the description, the script fails on zsh on my mac otherwise.
```
 make test
Checking system information
GOOS: darwin
GOARCH: amd64
System information checks passed.
Starting unit and integration tests
./hack/runtests.sh: line 31: @: unbound variable
make: *** [test] Error 1
```